File: java/EqualString.java
import java.util.Scanner;

public class EqualString {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        System.out.print("Enter first text: ");
        String first = scanner.nextLine();
        System.out.print("Enter second text: ");
        String second = scanner.nextLine();
        scanner.close();

        boolean equal = true;
        if(first.length() != second.length()){
            equal = false;
        }
        if(equal){
            for(int i = 0; i < first.length(); i++){
                char ch1 = first.charAt(i);
                char ch2 = second.charAt(i);
                if(ch1 != ch2){
                    equal = false;
                    break;                   
                }
            }
        }
        if(equal){
            System.out.println("equal texts");
        }else{
            System.out.println("not equal texts");
        }

    }
}

File: kt/EqualString.kt
import java.util.Scanner

fun main() {
    
        val scanner = Scanner(System.`in`)
        print("Enter first text: ")
        var first = scanner.nextLine()
        print("Enter second text: ")
        var second = scanner.nextLine()

        // without using predefine method method
        var equal = true
        if(first.length != second.length){
            equal = false
        }
        if(equal){
            for(i in 0 until first.length){
                val ch1 = first.toCharArray()[i]
                val ch2 = second.toCharArray()[i]
                if(ch1 != ch2){
                    equal = false
                    break                   
                }
            }
        }
        if(equal){
            System.out.println("equal texts")
        }else{
            System.out.println("not equal texts")
        }
    
}